app/forms/work_form.rb

```diff
@@ -46,6 +46,7 @@ def persisted?
   validates :license, presence: true, on: :deposit
 
   attribute :work_type, :string
+  before_validation ->(work_form) { work_form.work_type = work_form.work_type.presence }
```

Comment:
You may want to consider a related tweak here, too: https://github.com/sul-dlss/hungry-hungry-hippo/pull/1000

Reply:
Done.